### PR TITLE
chore(code): add test for indented code blocks

### PR DESCRIPTION
**Pull request by kellyjosephprice**
### 🧰  Changes

Add test for indented code block. 

There's an open ticket for indented code blocks losing some indentation, but I believe it's fixed by #70.

https://app.asana.com/0/1178132843162889/1199186890192012/f

### CODE REVIEW

File: components/Callout/index.jsx [hunk 1]
```diff
@@ -4,7 +4,6 @@ const PropTypes = require('prop-types');
 const Callout = props => {
   let { children } = props;
   const { attributes, theme, title, icon } = props;
-  if (!(title || children)) return '';
```
Comment (kellyjosephprice) [inline components/Callout/index.jsx:7]: In a previous PR, I pushed the test coverage below the threshold. In trying to write some tests to increase coverage, I found this line. `children` is required in `propTypes` so this _should_ never be true.

Comment (rafegoldberg) [reply components/Callout/index.jsx:7]: lol "Remove impossible line"

File: __tests__/flavored-parsers.test.js [hunk 2]
```diff
@@ -128,5 +128,15 @@ describe('Parse RDMD Syntax', () => {
...
+
+    it('handles indented code blocks', () => {
+      const mdx = `
+\`\`\`
+  const shouldBeIndented = true;
+  if (shouldBeIndented) pass();
+\`\`\`
+`;
+      expect(process(mdx)).toMatchSnapshot();
+    });
```
Comment (rafegoldberg) [inline __tests__/flavored-parsers.test.js:140]: Should we add a test for actual indented code blocks (as described [here](https://www.markdownguide.org/basic-syntax/#code-blocks)) too?

**Review by rafegoldberg — APPROVED** [review]
LGTM

**Comment by kellyjosephprice** [comment]
@rafegoldberg I went ahead and merged this, but please let me know if I should address anything.

**Comment by rafegoldberg** [comment]
## This PR was released!

🚀 Changes included in [<b>v6.23.0</b>](/readmeio/markdown/releases/tag/v6.23.0)
<br>

**Pull request merged**